apps/src/blockly/addons/cdoGenerator.ts

```diff
@@ -8,11 +13,9 @@ export default function initializeGenerator(blocklyWrapper) {
 
   // This function was a custom addition in CDO Blockly, so we need to add it here
   // so that our code generation logic still works with Google Blockly
-  blocklyWrapper.Generator.xmlToBlocks = function (name, xml) {
+  blocklyWrapper.Generator.xmlToBlocks = function (_name, xml) {
```

Comment:
Just so I can learn, what's the convention that prepends  _?

Reply:
It means the parameter is unused.

Reply:
I feel like we have a lot of unused parameters in Blockly, so this is useful for me to know; thanks, Molly!